packages/plugins/src/components/plugin-context/index.tsx

```diff
@@ -34,6 +35,8 @@ export function usePluginContext() {
  * A Higher Order Component used to inject Plugin context to the
  * wrapped component.
  *
+ * @deprecated Use `usePluginContext` hook instead.
```

Comment:
It mostly applies to WordPress context, so adding the version number could be useful:
```suggestion
 * @deprecated 6.8.0 Use `usePluginContext` hook instead.
```